controllers/commentsController/updateSingleCommentFlags.js

```diff
@@ -26,7 +26,7 @@ const updateSingleCommentFlags = async (req, res, next) => {
     const { applicationId } = req.token;
 
     if (!mongoose.Types.ObjectId.isValid(commentId)) {
-      return next(new CustomError(422, "invalid ID"));
+      return next(new CustomError(422, "Validation Error"));
```

Comment:
did you make this change just for string message differences..?
this change is not required

Reply:
it was the required error message as described by the  swagger docs